radix-transactions/src/model/executable.rs

```diff
@@ -62,14 +62,14 @@ impl From<(BlueprintId, GlobalAddress)> for PreAllocatedAddress {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq, ScryptoSbor, ManifestSbor)]
-pub struct TransactionCostingParameters {
+pub struct TransactionCostingParametersV2 {
```

Comment:
These structs do not use our conventional "versioning infra" (the macros, the wrappers, the traits, etc), so maybe we shouldn't use the `...V2` naming here.
Maybe something like "input costing params" vs "output costing params", or "receipt costing params"?
(unless this was already discussed and you are sure we want V1/V2)

Reply:
Sounds good to me, will switch the naming to indicate that our versioning infra is not used here.